src/patternfly/components/DataList/examples/data-list-width-example.hbs

```diff
@@ -12,15 +12,15 @@
     {{#> data-list-cell}}
       <div class="Preview__placeholder">
         <b>default</b>
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>      
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
       </div>
     {{/data-list-cell}}
   {{/data-list-item}}
 {{/data-list}}
 
 {{!-- Example 2 --}}
 <h2 class="Preview__section-title">Flex modifiers - example 2</h2>
-{{#> data-list data-list--modifier="pf-u-box-shadow-md" data-list--attribute='aria-label="Width modifier data list example 2"'}}
```

Comment:
if we remove this do we want to add it back via the component styles or was it not supposed to be there at all?

Reply:
Wasn't supposed to be there at all. If the data list needs a box shadow, you can put it in a card. There's an issue open to create that as a demo - https://github.com/patternfly/patternfly-next/issues/841